templates/partial/search_result.html

```diff
@@ -37,7 +38,7 @@ <h3 class="govuk-heading-m govuk-!-margin-bottom-2">
         <div class="govuk-summary-list__row">
           <dt class="govuk-summary-list__key">Last updated date</dt>
           <dd class="govuk-summary-list__value">
-            {{result.last_updated}}
+            {{result.last_updated|naturaltime}}
```

Comment:
or `{{result.last_updated}} ({{result.last_updated|naturaltime}})` ?

e.g. **"Last updated date: 25th January 2023 (one week ago)"**

So the actual date is still visible. Ignore if I've not understood how the django pipe works!